testcases/two_zone_apartment_hydronic/models/TwoZoneApartmentHydronic/Components/Rooms/thermalZone.mo

```diff
@@ -0,0 +1,859 @@
+  Buildings.Fluid.Sensors.Density density(redeclare package Medium = MediumA,
+      warnAboutOnePortConnection=false)
+    "Air density inside the building"
+    annotation (Placement(transformation(extent={{-34,-20},{-44,-10}})));
+  Modelica.Thermal.HeatTransfer.Sensors.TemperatureSensor TRooAirSen
+    "Room air temperature"
+    annotation (Placement(transformation(extent={{66,18},{76,28}})));
+  Modelica.Blocks.Math.MultiSum multiSum(nu=1)
+    annotation (Placement(transformation(extent={{-64,2},{-56,10}})));
+  Modelica.Blocks.Interfaces.RealOutput TRooAir "Room air temperature"
+    annotation (Placement(transformation(extent={{160,28},{180,48}}),
+        iconTransformation(extent={{160,28},{180,48}})));
+  Buildings.Fluid.Sources.MassFlowSource_WeatherData
+                                           souInf(
+    redeclare package Medium = MediumA,
+    m_flow=1,
+    use_m_flow_in=true,
+    use_C_in=true,
+    nPorts=1) "source model for air infiltration"
+    annotation (Placement(transformation(extent={{8,0},{16,8}})));
+  Modelica.Blocks.Math.Gain gain(k=-1)
+    annotation (Placement(transformation(extent={{-16,2},{-8,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_a supplyAir(redeclare final package
+      Medium = MediumA) "Supply air"
+    annotation (Placement(transformation(extent={{-110,22},{-90,42}}),
+        iconTransformation(extent={{-110,22},{-90,42}})));
+  Modelica.Fluid.Interfaces.FluidPort_b returnAir(redeclare final package
+      Medium = MediumA) "Return air"
+    annotation (Placement(transformation(extent={{-110,-30},{-90,-10}}),
+        iconTransformation(extent={{-110,-30},{-90,-10}})));
+  TwoZoneApartmentHydronic.Components.BaseClasses.InternalLoad lig(
+    radFraction=0.5,
+    latPower_nominal=0,
+    senPower_nominal=1.5)
+    annotation (Placement(transformation(extent={{-100,48},{-88,60}})));
+  TwoZoneApartmentHydronic.Components.BaseClasses.InternalLoad equ(
+    latPower_nominal=0,
+    senPower_nominal=4,
+    radFraction=0.5)
+    annotation (Placement(transformation(extent={{-100,72},{-88,84}})));
+  TwoZoneApartmentHydronic.Components.BaseClasses.OccupancyLoad occ(
+    radFraction=0.5,
+    co2Gen=8.64e-6,
+    occ_density=1/Afloor,
+    senPower=60,
+    latPower=20)
+    annotation (Placement(transformation(extent={{-100,60},{-88,72}})));
+  Modelica.Blocks.Math.MultiSum sumRad(nu=3) "Sum of radiant internal gains"
+    annotation (Placement(transformation(extent={{-48,76},{-42,82}})));
+  Modelica.Blocks.Math.MultiSum sumCon(nu=3) "Sum of convective internal gains"
+    annotation (Placement(transformation(extent={{-50,66},{-44,72}})));
+  Modelica.Blocks.Math.MultiSum sumLat(nu=3) "Sum of latent internal gains"
+    annotation (Placement(transformation(extent={{-48,56},{-42,62}})));
+  IBPSA.Utilities.IO.SignalExchange.Read reaPLig(y(unit="W"), description=
+        "Lighting power submeter",
+    KPIs=IBPSA.Utilities.IO.SignalExchange.SignalTypes.SignalsForKPIs.ElectricPower)

```

Comment:
The [design documentation](https://ibpsa.github.io/project1-boptest/docs-design/core_kpi.html#energy-use-in-a-given-period-of-time) and [journal article](https://www.tandfonline.com/doi/full/10.1080/19401493.2021.1986574) specify that the energy, cost, and emission KPIs are for HVAC site energy.  Lighting and plug load submeters are fine to output I think, but the ``KPIs`` parameter should not be assigned to ``ElectricPower`` for inclusion in energy KPI calculations.  Instead, they should just be ``None``.

Reply:
removed ElectricPower tag from lighting and plug loads